go.mod

```diff
@@ -5,7 +5,7 @@ go 1.18
 require (
 	github.com/AdaLogics/go-fuzz-headers v0.0.0-20210715213245-6c3934b029d8
 	github.com/Microsoft/go-winio v0.5.2
-	github.com/Microsoft/hcsshim v0.9.8
+	github.com/Microsoft/hcsshim v0.9.10
```

Comment:
q: hcsshim imports containerd/containerd right? Wont this cause a cyclic dependency issue?

Reply:
No. The osversion package in hcsshim where these new functions were added do no depend on any pkg from containerd. I don't think cyclic dependency should be a concern here.

Reply:
this change is no worse than where we were with the prior imports.. perhaps at a later date we would request decoupling of the system level api... perhaps by inclusion into golang..

Cheers